CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
 # osquery Changelog
 
+<a name="5.13.0"></a>
+## [5.13.0](https://github.com/osquery/osquery/releases/tag/5.13.0)
+
+[Git Commits](https://github.com/osquery/osquery/compare/5.12.2...5.13.0)
+
+Representing commits from 20 contributors! Thank you all.
+
+### Table Changes
+
+- The Python manifest directories, `.egg-info` and `.dist-info`, contain flat file hierarchies ([#8318](https://github.com/osquery/osquery/pull/8318))
+- Add new file path for `alf` plist (macOS 15) ([#8352](https://github.com/osquery/osquery/pull/8352))
+- Table `users` on linux by default to return only users in `/etc/passwd` ([#8342](https://github.com/osquery/osquery/pull/8342))
+- Add `sha256` hash to `apparmor_profiles` table ([#8345](https://github.com/osquery/osquery/pull/8345))
+- Add support for metalink and store repo config file name in `yum_sources` table ([#8307](https://github.com/osquery/osquery/pull/8307))
+- Update `user_ssh_keys` with additional details for OpenSSL-style keys ([#8314](https://github.com/osquery/osquery/pull/8314))
+- Fix table `dns_resolvers` dns-search bug with multiple search domains ([#8329](https://github.com/osquery/osquery/pull/8329))
+- Fix `process_open_sockets` to correctly displays `family` and `protocol` on macOS ([#8315](https://github.com/osquery/osquery/pull/8315))
+- Add missing SSH key types to `authorized_keys` that support FIDO2 authentication ([#8319](https://github.com/osquery/osquery/pull/8319))
+
+### Under the Hood improvements
+
+- Improve error message when required constraint missing ([#8358](https://github.com/osquery/osquery/pull/8358))
+- Add verbose logging when distributed requests fail and retry ([#8321](https://github.com/osquery/osquery/pull/8321))
+
+### Bug Fixes
+
+- Fix for Potential memory leak in class `ServiceArgumentParser`'s Constructor ([#8368](https://github.com/osquery/osquery/pull/8368))
+- Fix for Crash in `ServiceArgumentParser` via `ServiceMain` ([#8353](https://github.com/osquery/osquery/pull/8353))
+- Fixing real precision by limiting precision to 15 digits ([#8355](https://github.com/osquery/osquery/pull/8355) and [#8302](https://github.com/osquery/osquery/pull/8302))
+- Make fallthrough explicit in `sqlite_encoding.cpp` ([#8361](https://github.com/osquery/osquery/pull/8361))
```

Comment:
I would move this under `Build` since it's just to not warn at compile time when enabling a certain warning flag.